src/objects/page/page.scss

```diff
@@ -3,16 +3,16 @@
  *
  * 1. Avoid grid blowouts by specifying the `min-width` of the column.
  *    @see https://css-tricks.com/you-want-minmax10px-1fr-not-1fr/
- * 2. Attempts to fill the available area (assuming `height: 100%` is applied
- *    to all parent elements) & keep the footer at the bottom of that area.
- *    @see https://css-tricks.com/couple-takes-sticky-footer/#there-is-grid
+ * 2. Attempts to fill the available area
+ *    & keep the footer at the bottom of that area.
+ *    @see https://css-tricks.com/couple-takes-sticky-footer/#aa-there-is-grid
  */
 
 .o-page {
   display: grid;
   grid-template-columns: minmax(0, 1fr); // 1
   grid-template-rows: 1fr auto; // 2
-  min-block-size: 100%;
+  min-block-size: 100vh;
```

Comment:
Will this result in short pages pushing the footer beneath the browser chrome in iOS Safari?

Reply:
Oh I should check that, good point

Reply:
Yes it does. Hmm.

Off the top of your head do you know any other way to fix the double scrollbar issue? I can fiddle with it more if not.

Reply:
Found a solution